iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/checks/ContainerPrivilegedModeCheck.java

```diff
@@ -32,16 +32,21 @@ public class ContainerPrivilegedModeCheck extends AbstractKubernetesObjectCheck
 
   @Override
   void registerObjectCheck() {
+    checkOnPrivilegedModeWithKey("containers");
+    checkOnPrivilegedModeWithKey("initContainers");
+  }
+
+  private void checkOnPrivilegedModeWithKey(String key) {
     register("Pod", pod ->
-      pod.blocks("containers").forEach(container ->
+      pod.blocks(key).forEach(container ->
         container.block("securityContext")
           .attribute("privileged")
             .reportIfValue(isTrue(), MESSAGE)
```

Comment:
_minor_ let's fix the alignment here (there is an extra space at the beginning). Same for the other register.